docs/reference/modules/doc-type.md

```diff
@@ -0,0 +1,135 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/doc-type`
+
+This module establishes the basic structure and functionality for all content documents, including pages and pieces. More specifically, it is the foundation for piece *types* and page *types*. The features below are available on all piece and page types except where they are overridden in the `@apostrophecms/piece-type` and `@apostrophecms/page-type` modules.
+
+This module is almost never configured directly in Apostrophe projects. The only reason to configure this module directly would be to apply the changes to *every* page type and piece type, including those Apostrophe core (e.g., `@apostrophecms/user`, `@apostrophecms/home-page`).
+
+**Extends:** `{{ $frontmatter.extends }}`
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`autopublish`](#autopublish) | Boolean | Set to `true` to publish all saved edits immediately. |
```

Comment:
I thought about this a bit. It's a weird position because some of these options (and methods, I think) are not applicable to pages, but come from this module.

Reply:
I'll remove autopublish here since it's covered in the piece type page.